source/architecture.rst

```diff
@@ -67,16 +63,134 @@ converging to a reviewed architectural proposition as described on :numref:`new_
 .. figure:: images/new_architecture.png
     :width: 100%
     :align: center
-    :alt: Revised dojot Architecture
+    :alt: Revised *dojot* Architecture
 
     Revised Architecture
 
 More detailed and updated information can be found on the `dojot Github repository <https://github.com/dojot>`_.
 
+Each one of the components that are part of the architecture are briefly described on the sub-sections below.
+
+Kafka + Subscription Manager + NGSI
+***********************************
+
+Using Apache Kafka in conjunction with a subscriptions managers and a NGSI interface translator we compose a context
+broker analogous to Fiware Orion, with the addition of functionalities that are tipical to brokers, more processing
+power and scalability. Apache Kafka is a distributed platform that was consolidated for the construction of data
+pipelines in real time and streaming applications, differently from the Fiware Orion core that is a non-relational DB.
+
```

Comment:
I would change this paragraph to:
"Apache Kafka is a distributed messaging platform that can be used by applications which need to stream data or consume/produce data pipelines. In contrast to what Orion is, Kafka seems to be more appropriate to fulfil dojot's architectural requirements (responsibility isolation, simplicity, and so on). And using it with a subscription manager and a NGSI interface translator, we can compose an entity which is very close to the features offered by Orion, in addition to improved speed and easier scalability."

Reply:
Yeah, much better